force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -69,7 +79,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        if (b == 0) {
```

Comment:
Wouldn't hurt to add a null check to the beginning of this method too so that you don't run into an error if either value is null.